cmd/cluster/dynatrace/rootCmd.go

```diff
@@ -0,0 +1,26 @@
+package dynatrace
+
+import (
+	"github.com/openshift/osdctl/pkg/k8s"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const (
+	DynatraceTenantKeyLabel    string = "sre-capabilities.dtp.tenant"
+	HypershiftClusterTypeLabel string = "ext-hypershift.openshift.io/cluster-type"
+)
+
+func NewCmdDynatrace(streams genericclioptions.IOStreams, client *k8s.LazyClient) *cobra.Command {
```

Comment:
**chore:** if I'm not mistaken, these two parameters are unused, can we remove them?